workspaces/src/network/sandbox.rs

```diff
@@ -206,3 +234,106 @@ impl Sandbox {
         Ok(())
     }
 }
+
+#[cfg(feature = "experimental")]
+impl Sandbox {
+    pub(crate) async fn changes_in_block(&self, block_reference: BlockReference) -> Result<()> {
+        // NOTE: RpcStateChangesInBlockByTypeResponse is an empty struct with no fields, so don't do anything with it.
+        self.client()
+            .query_nolog(&RpcStateChangesInBlockRequest { block_reference })
+            .await
+            .map_err(|e| SandboxErrorCode::ChangesInBlockFailure.custom(e))?;
+
+        Ok(())
+    }
+
+    pub(crate) async fn changes(
+        &self,
+        block_reference: BlockReference,
+        state_changes_request: StateChangesRequestView,
+    ) -> Result<RpcStateChangesInBlockResponse> {
+        let resp = self
+            .client()
+            .query_nolog(&RpcStateChangesInBlockByTypeRequest {
+                block_reference,
+                state_changes_request,
+            })
+            .await
+            .map_err(|e| SandboxErrorCode::ChangesFailure.custom(e))?;
+
+        Ok(resp)
+    }
+
+    pub(crate) async fn check_tx(
+        &self,
+        signed_transaction: SignedTransaction,
+    ) -> Result<RpcBroadcastTxSyncResponse> {
+        let resp = self
+            .client()
+            .query_nolog(&RpcCheckTxRequest { signed_transaction })
+            .await
+            .map_err(|e| SandboxErrorCode::CheckTxFailure.custom(e))?;
+
+        Ok(resp)
+    }
+
+    // This method is a stub and does nothing.
+    pub(crate) async fn genesis_config(&self) -> Result<()> {
+        // NOTE: RpcGenesisConfigResponse is an empty struct with no fields, so don't do anything with it.
```

Comment:
Hmm, this does not sound true: https://docs.rs/near-jsonrpc-client/latest/near_jsonrpc_client/methods/EXPERIMENTAL_protocol_config/type.RpcProtocolConfigResponse.html it is clearly not an empty struct.

Reply:
Missed that, I'll update